packages/octane-blueprint-utils/index.js

```diff
@@ -21,7 +21,7 @@ function getEmberCLIVersionFromTarball() {
     });
 
     got
-      .stream('http://github.com/ember-cli/ember-cli/tarball/master')
```

Comment:
do all repositories have tarballs?

Reply:
is this what you get when you click "Download" instead of clone?

Reply:
Honestly, no idea; Robert implemented it but I tested and it looks to be working.

Reply:
I think you need to breach > 50/reqs a minute to see if it works?

Reply:
@NullVoxPopuli  Didn't you know that I worked for a Load testing company ;). 

I meant I tested the `getRepoVersion` function. As you pointed, `getEmberCLIVersionFromTarball ` was likely not executed.

Reply:
@NullVoxPopuli tarball download is not throttled